controllers/storagecluster/cephobjectstores_test.go

```diff
@@ -12,14 +13,23 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
+type CephObjectStoreTestCase struct {
+	label                string
+	createRuntimeObjects bool
+	gatewayInstances     int32
+}
+
 func TestCephObjectStores(t *testing.T) {
-	var cases = []struct {
-		label                string
-		createRuntimeObjects bool
-	}{
+	var cases = []CephObjectStoreTestCase{
+		{
+			label:                "case 1", // Ensure that cephObjectStores are created on non-cloud Platform with default values
+			createRuntimeObjects: false,
+			gatewayInstances:     1,
+		},
 		{
-			label:                "case 1", // Ensure that cephObjectStores are created on non-cloud Platform
+			label:                "case 2", // Ensure that cephObjectStores are created on non-cloud Platform with configured values
 			createRuntimeObjects: false,
```

Comment:
I see that `createRuntimeObjects` is false in both cases. If that's the case can not we get rid of this value?

Reply:
I thought that was something of a curiosity myself. It seems to be standard across the Ceph storage interface test case structs, though, and there is some value in consistency for ease of modification of all three tests when it comes up. I certainly don't have strong feelings on this either way, though it's unrelated to the current change set.